src/main/java/beam/physsim/jdeqsim/AgentSimToPhysSimPlanConverter.java

```diff
@@ -264,8 +262,6 @@ private void setupActorsAndRunPhysSim(IterationEndsEvent iterationEndsEvent) {
         completableFutures.add(CompletableFuture.runAsync(() -> physsimNetworkEuclideanVsLengthAttribute.notifyIterationEnds(iterationNumber)));
 
         writeIterationCsv(iterationNumber);
-        Road.setAllRoads(null);
```

Comment:
Why was this removed?

Reply:
This line clears a static field for I suppose freeing the memory. Because that static field is removed (that field is moved to Vehicle) we don't more need to explicitly clear it. It is disposed by GC when it frees memory automatically.